subprojects/launcher/src/integTest/groovy/org/gradle/launcher/continuous/ArchivesContinuousIntegrationTest.groovy

```diff
@@ -123,6 +125,7 @@ class ArchivesContinuousIntegrationTest extends Java7RequiringContinuousIntegrat
     }
 
     @Ignore("inputs from resources are ignored")
+    @Unroll
```

Comment:
I missed the `@Ignore` - so it has no effect at all. I only saw that the test method was using `#` in its method name and thought that it should probably be unrolled. I can also remove the `@Unroll` again, WDYT?